### PR TITLE
starting staging regression suite

### PR DESCRIPTION
**Pull request by Bradern8t0r**
I am starting the staging regression suite. I have the AO login, BO login and the corporate test. 

Due to differences between staging and production, I had to gut these tests. We will revisit them in the future.

### CODE REVIEW

File: demo/STAGE_regressionSuite/tests/STAGE_corp_business.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var i = require('../../tools.js')
+var stage = require('../STAGE_tools.js')
+
+
+var expectedTitle = 'Protect Your People | LegalShield'
+var exTitleCheckout = 'Legalshield - Checkout'
```
Comment (skylarburris) [inline demo/STAGE_regressionSuite/tests/STAGE_corp_business.js]: is the 'S' supposed to be capitalized?

Comment (Bradern8t0r) [reply demo/STAGE_regressionSuite/tests/STAGE_corp_business.js]: Yes. It is correct.

File: demo/STAGE_regressionSuite/tests/STAGE_ao-login.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 var i = require('../../tools.js')
 var stage = require('../STAGE_tools.js')
+var wait = require('../../checkout_tools.js')
+
```
Comment (TrentonWalkup) [inline demo/STAGE_regressionSuite/tests/STAGE_ao-login.js]: extra space

File: demo/STAGE_regressionSuite/tests/STAGE_ao-login.js [hunk 3]
```diff
@@ -11,18 +13,18 @@ what('logging into ao successfully', function() {
...
-  })
+    wait.waitClick('[name*="login"]')
+    wait.waitAssertTitle(expectedTitle)
+    browser.pause(20000)
+    wait.waitClick('.user-pic')
+    wait.waitClick('div.logout a')
+    browser.pause(2500)
+    })
 what('logging into ao unsuccessfully', function() {
   switchToFrame("loginModal")
```
Comment (TrentonWalkup) [inline demo/STAGE_regressionSuite/tests/STAGE_ao-login.js]: let's try to keep our stuff aligned correctly

File: demo/STAGE_regressionSuite/tests/STAGE_bo-login.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+var i = require('../../tools.js')
+var stage = require('../STAGE_tools.js')
+var wait = require('../../checkout_tools.js')
+
```
Comment (TrentonWalkup) [inline demo/STAGE_regressionSuite/tests/STAGE_bo-login.js]: extra space

File: demo/STAGE_regressionSuite/tests/STAGE_bo-login.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+var expectedLoginMessage = 'Welcome, BROKER ONLY LS IT';
+var expectedBadloginmessage = '*Notice: Username and/or Password incorrect. Please try again.'
+var expectedMystatisticsmessage = 'My Statistics'
+var expectedReportsmessage= 'Active Member Report Menu'
+var expectedResourcesmessage = 'Customize My Website'
+var expectedLogoutmessage = 'WHY LEGALSHIELD'
```
Comment (TrentonWalkup) [inline demo/STAGE_regressionSuite/tests/STAGE_bo-login.js]: if we're not asserting these why are they here

**Comment by robertvazquezlegalshield** [comment]
LGTM

**Comment by Bradern8t0r** [comment]
Approved by @TrentonWalkup and @robertvazquezlegalshield.

**Pull request merged**